openslides/core/static/js/core/site.js

```diff
@@ -355,10 +355,27 @@ angular.module('OpenSlidesApp.core.site', [
                                     alreadyConverted.push(stack);
                                     break;
                                 case "img":
+                                    // pixel a4 in 72dpi: 595 x 842 pixel
+                                    // TODO: proper way to calculate the space left on the page.
+                                    // This requires further information
```

Comment:
basically this is a todo comment to enhance it when more information about the document is present. All the information is required.
The comment also is meant to explain why we use 590 and 840.